content/articles/nodejs-network-requests-using-axios/index.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /nodejs-network-requests-using-axios/
+title: Node.js Network requests using Axios
+description: This article goes through Axios and how to use it to make network request in Node.js. Axios is a very popular javascript framework used to perform network requests.
+author: peter-kayere
+date: 2021-05-31T00:00:00-14:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/nodejs-network-requests-using-axios/hero.jpg
+    alt: Node.js Network requests using Axios
+---
+
+### Introduction
+
+In the modern technological world, it is very rear to find stand-alone applications. Most applications depend on resources from other applications. These resources are made available through the use of APIs. Servers are no exception. Though servers store their own relevant data, they sometimes need data from other servers. To access the data, servers need a way to communicate with the API(s) of the depended servers. This is where Axios comes in.
+
+Axios is a very popular javascript framework used to perform network requests. Axios works both on the browser and Node.js runtime. Axios is promise based but also allows the modern async/await methods. This article goes through Axios and how to use it to make network request in Node.js.
+
+### Prerequisites
+
+To follow through this article you need to have Node.js installed and understand its basics. How to set up a simple server, and how to do some basic configuration. Go through [this]() article to get yourself up to speed.
+
+Through this article, we are going to make network requests to a free dogs api from our Node.js application. Our application will be sending back the response it receives from the api.
+
+Let's get started!
+
+### Setting up our server
+
+Start by creating a new Node.js project. Create a folder with the name of your choice and run the following commands from the terminal.
+
+```bash
```

Comment:
If you could move the comments out of the code snippet since people tend to copy the commands. 
It will also add more content to the article...

Reply:
Done